src/applications/personalization/profile/components/direct-deposit/BankInfoCNP.jsx

```diff
@@ -176,20 +176,29 @@ export const BankInfoCNP = ({
 
   // When direct deposit is not set up, we will show
   const notSetUpContent = (
-    <button
-      className="va-button-link"
-      ref={editBankInfoButton}
-      onClick={() => {
-        recordEvent({
-          event: 'profile-navigation',
-          'profile-action': 'add-link',
-          'profile-section': 'cnp-direct-deposit-information',
-        });
-        toggleEditState();
-      }}
-    >
-      Please add your bank information
-    </button>
+    <div className={classes.bankInfo}>
+      <dl className="vads-u-margin-y--0 vads-u-line-height--6">
+        <dt className="sr-only" />
+        <dd>Please add your bank information</dd>
```

Comment:
We probably want to keep this "empty" state consistent with the contact info section and have this read "Edit your profile to add your bank information". I'll ask Liz on the front page of this PR.